app/waf/src/controllers/adc.controller.ts

```diff
@@ -87,119 +80,69 @@ export class AdcController extends BaseController {
     reqBody.tenantId = await this.tenantId;
 
     //TODO: Reject create ADC HW request with duplicated mgmt IP address
-    if (reqBody.type === 'HW') {
-      await this.trustAdc(reqBody);
-    }
-
-    try {
-      return new Response(Adc, await this.adcRepository.create(reqBody));
-    } catch (e) {
-      throw new HttpErrors.UnprocessableEntity(e.message);
-    }
-  }
-
-  async trustAdc(adc: Partial<Adc>): Promise<void> {
-    if (!adc.management || !adc.management.ipAddress) {
-      throw new HttpErrors.BadRequest(
-        'IP address and admin passphrase are required to trust ADC hardware',
-      );
-    }
-
-    let body: TrustedDeviceRequest = {
-      devices: [
-        {
-          targetHost: adc.management.ipAddress,
-          targetPort: adc.management.tcpPort || 443,
-          //TODO: Need away to input admin password of BIG-IP HW
-          targetUsername: 'admin',
-          targetPassphrase: 'admin',
-        },
-      ],
-    };
-
-    let devices: TrustedDevice[];
-
+    let adc: Adc;
     try {
-      devices = (await this.trustedDeviceService.trust(
-        ASG_HOST,
-        ASG_PORT,
-        body,
-      )).devices;
+      adc = await this.adcRepository.create(reqBody);
     } catch (e) {
-      //TODO: Request TrustedDevices to hide admin passphrase in error message
       throw new HttpErrors.UnprocessableEntity(e.message);
     }
 
-    //Wait until trusted state become ACTIVE
-    let retries = 0;
-    while (retries < 15) {
-      if (devices.length !== 1) {
-        throw new HttpErrors.UnprocessableEntity(
-          'Trusting device fails: Response size is ' + devices.length,
+    if (adc.type === 'HW') {
```

Comment:
Let it be here for a little while. At least we need this to do ASG testing. If we really hope to remove it before releasing the product, we can do it at that time.

Reply:
It's OK. 
Suppose the trust functionality would soon become stable. And it will be tested every time in setupOn, removing it can make the code clean.

Reply:
Agree to disable HW path, after our VE onboarding path is stable.